docs/CHANGELOG.asciidoc

```diff
@@ -10,12 +10,105 @@
 
 Review important information about the {kib} 8.0.0 releases.
 
+* <<release-notes-8.0.0-rc2>>
 * <<release-notes-8.0.0-rc1>>
 * <<release-notes-8.0.0-beta1>>
 * <<release-notes-8.0.0-alpha2>>
 * <<release-notes-8.0.0-alpha1>>
 
 --
+[[release-notes-8.0.0-rc2]]
+== {kib} 8.0.0-rc2
+
+coming::[8.0.0-rc2]
+
+For information about the {kib} 8.0.0-rc2 release, review the following information.
+
+[float]
+[[breaking-changes-8.0.0-rc2]]
+=== Breaking change
+
+Breaking changes can prevent your application from optimal operation and performance.
+Before you upgrade, review the breaking change, then mitigate the impact to your application.
+
+// tag::notable-breaking-changes[]
+
+[discrete]
+[[breaking-122722]]
+.Removes the ability to use `elasticsearch.username: elastic` in production
+[%collapsible]
+====
+*Details* +
+In production, you are no longer able to use the `elastic` superuser to authenticate to {es}. For more information, refer to {kibana-pull}122722[#122722].
+
+*Impact* +
+When you configure `elasticsearch.username: elastic`, {kib} fails.
+====
+  
+// end::notable-breaking-changes[]
+
+To review the breaking changes in previous versions, refer to the following: 
+
+<<breaking-changes-8.0.0-rc1,8.0.0-rc1>> | <<breaking-changes-8.0.0-beta1,8.0.0-beta1>> | <<breaking-changes-8.0.0-alpha2,8.0.0-alpha2>> | 
+<<breaking-changes-8.0.0-alpha1,8.0.0-alpha1>>
+ 
+[float]
+[[features-8.0.0-rc2]]
+=== Features
+{kib} 8.0.0-rc2 adds the following new and notable features.
+
+Dashboard::
+Dashboard Integration {kibana-pull}115991[#115991]
+Elastic Security::
+For the Elastic Security 8.0.0-rc2 release information, refer to {security-guide}/release-notes.html[_Elastic Security Solution Release Notes_].
+Machine Learning::
+Adds the new Model Management tab {kibana-pull}115772[#115772]
+Management::
+Display managed badge for transforms {kibana-pull}117679[#117679]
```

Comment:
[This PR](https://github.com/elastic/kibana/pull/117679) was already published in the [rc1 release notes](https://www.elastic.co/guide/en/kibana/8.0/release-notes-8.0.0-rc1.html). This seems like a bug that it was picked up here too
```suggestion

```